src/amp-story-player/amp-story-player-impl.js

```diff
@@ -188,9 +189,16 @@ const LOG_TYPE = {
 };
 
 /**
- * Flag to show or hide the desktop panels player experiment.
- * @const {boolean}
+ * Different UI experiences to display the story.
+ * @const @enum {number}
  */
+export const UIType = {
+  MOBILE: 0,
+  DESKTOP_PANELS: 1, // Default desktop UI displaying previous and next pages.
+  DESKTOP_FULLBLEED: 2, // Desktop UI if landscape mode is enabled.
+  DESKTOP_ONE_PANEL: 4, // Desktop UI with one panel and space around story.
+  VERTICAL: 3, // Vertical scrolling versions, for search engine bots indexing.
+};
```

Comment:
Can we import this instead of copy/pasting? If we can't import for any reason, I'd be OK with using the Numbers directly in the code, like
`uiTypeNumber ===  2 /** DESKTOP_FULLBLEED */`

Having duplicated enums is confusing for people with no context and often leads to bugs :(